tests/integration/heartbeat/heartbeat.spec.ts

```diff
@@ -109,10 +109,11 @@ describe('heartbeat', function () {
 
         await requestSender.pulse(id);
         const response = await requestSender.getHeartbeat(id);
-        expect(response.body.id).toBe(id);
+        expect(response.body).toHaveProperty('id', id);
+        expect(response.body).toHaveProperty('lastHeartbeat');
         expect(new Date(response.body.lastHeartbeat)).toBeInstanceOf(Date);
```

Comment:
This expect will always be true!

Reply:
I want to ensure that the field `lastHeartbeat` is a Date. Because `response.body` is a JSON, I can't just expect `lastHeartbeat` to be an instance of Date. If it's not a valid date, `new Date` won't create a Date and the expect will fail.

Reference: https://stackoverflow.com/a/72507679

Reply:
See the comment inside it, it won't work..
You can test it yourself [here](https://www.typescriptlang.org/play/?#code/MYewdgzgLgBAhjAvDMBTA7jAInKqAUOeAdGCOvgJSUDcAUKJLAEZIoba4EBEAZgCYRutBuAggANqmISQAc3wIAlkzhhgqEL055aMAPT6YAdSUSJMAA4AnFbG5RrAV1TdRkSdNkLWK6Go0tHVQ9QxMzCxs7GAdnVyA)

Reply:
I've removed this line.